src/_guides/testing.md

```diff
@@ -130,14 +130,11 @@ For details, see the
 
 Consider using continuous integration (CI) to build your project
 and run its tests after every commit. Two CI services for GitHub are
-[Travis CI](https://travis-ci.org/) (for OS X and Unix) and
+[Github Actions](https://github.com/features/actions) and
 [AppVeyor](https://www.appveyor.com/) (for Windows).
```

Comment:
It looks like AppVeyor supports Windows, Linux, macOS, just like GitHub Actions does, so let's drop the " (for Windows)".